packages/nodes-base/nodes/Wordpress/GenericFunctions.ts

```diff
@@ -30,6 +30,7 @@ export async function wordpressApiRequest(
 		qs,
 		body,
 		uri: uri || `${credentials.url}/wp-json/wp/v2${resource}`,
+		rejectUnauthorized: !credentials.allowUnauthorizedCerts,
```

Comment:
isn't this already handled in credentials by `skipSslCertificateValidation: '={{$credentials.allowUnauthorizedCerts}}',` ?

Reply:
`skipSslCertificateValidation` only applies for the credential test, this sets it for the other node requests.